src/containers/MyApplications.jsx

```diff
@@ -11,9 +11,8 @@ const mapStateToProps = (state, ownProps) => ({
 })
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
-  fetchApps: () => {
-    dispatch(fetchApps())
-  }
+  fetchApps: () => dispatch(fetchApps()),
+  uninstallApp: (appSlug) => dispatch(uninstallApp(appSlug))
```

Comment:
IMHO `uninstallApp` should not have parameter as the whole component is contextualized with an `appSlug`. What do you think ?
```javascipt
uninstallApp: () => dispatch(uninstallApp(ownProps.appSlug))
```

Reply:
In fact, the `appSlug` is grabbed from the router, so it's directly available in the props like that. It's why I use it as parameter here.